### DISCUSSION TITLE
Other attributes for almost all hostile mobs.

### DISCUSSION THREAD

**Discussion by khanh3993** [PurpurMC/Purpur#1536] [Feature Requests]
Like the title, at this time purpur have an attribute option for all mobs, but no other than max_health.
Can devs add some attributes for hostile mobs: `generic.attack_damage`,  `generic.movement_speed`, `generic.follow_range`.
Hostile mobs list: _Zombie_, _Zombie Villager_, _Creeper_, _Wither Skeleton_, _Vindicator_, _Sliverfish_, _Spider_, _Cave Spider_, _Ravager_, _Husk_, _Endermite_, _Hoglin_, _Zoglin_, _Piglin_, _Piglin Brute_

**Comment by granny** [comment]
This is two feature requests in one. Please move one of these into it's own discussion post.

**Reply by khanh3993** [reply]
Edited

**Comment by khanh3993** [comment]
Updated list mobs.
